subt_ros/launch/competition_init.launch

```diff
@@ -40,10 +40,11 @@
       name="set_pose_relay"/>
   </group>
 
+  <!-- A rosbag recorder is in the subt_ros/src/SubtRosRelay.cc -->
   <!-- rosbag a /robot_data topic which can be used by a team to record
-       custom data -->
+       custom data
   <node pkg="rosbag" type="record" name="rosbag_robot_data"
-        args="record --split --size=1000 --max-splits=2 -O $(env HOME)/.ros/robot_data.bag -e '/robot_data(.*)'"/>
+        args="record - -split - -size=1000 - -max-splits=2 -O $(env HOME)/.ros/robot_data.bag -e '/robot_data(.*)'"/>-->
```

Comment:
Nit: This could probably just be removed entirely at this point, since it will still be in the git history.

Reply:
Removed in 753292ec9e79b397ed1197e9fd5ba45e60d31750